test/features/environment.py

```diff
@@ -12,7 +15,7 @@
 BASE_URL = 'http://ckan:5000/'
 
 # URL of remote Chrome instance.
-REMOTE_CHROME_URL = 'http://chrome:4444/wd/hub'
+REMOTE_CHROME_URL = 'http://localhost:4444/wd/hub'
```

Comment:
This seems likely to break the tests. Why would it need to change?

Reply:
+

Reply:
Will be fixed and pushed by Alex shortly.